orchestra/frontend/src/state/slices/dashboardTasks.ts

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios'
```

Comment:
i can't remember what the slices are for?

Reply:
to the extent that we can make a short readme explaining our data model/how to create a new slice and selectors, that would be amazing.

Reply:
I think the slices mainly houses the actions available for that particular view or component, whereas the selectors act as the getters of the redux state.

Reply:
Let me write up a readme to clean this up!